robots/commenter/main.go

```diff
@@ -179,9 +186,19 @@ func main() {
 		c = github.NewDryRunClient(secretAgent.GetTokenGenerator(o.token), secretAgent.Censor, o.graphqlEndpoint, o.endpoint.Strings()...)
 	}
 
-	query, err := makeQuery(o.query, o.includeArchived, o.includeClosed, o.updated)
+	ctx := context.Background()
+	githubClient, err := createGithubClient(ctx, o.endpoint, o.token)
```

Comment:
IMHO, I would stick with the `new` idiom here:

```suggestion
	githubClient, err := newGithubClient(ctx, o.endpoint, o.token)
```